src/ChatGptNet/Models/ChatGptParameters.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Text.Json.Nodes;
+
+namespace ChatGptNet.Models;
+
+/// <summary>
+/// Represents chat completion parameters.
+/// </summary>
+public class ChatGptParameters
+{
+    /// <summary>
+    /// Gets or sets the temperature option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-temperature for more information
+    /// </remarks>
+    public double? Temperature { get; set; }
+
+    /// <summary>
+    /// Gets or sets the top_p option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-top_p for more information
+    /// </remarks>
+    public double? TopP { get; set; }
+
+    /// <summary>
+    /// Gets or sets the n option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-n for more information
+    /// </remarks>
+    public int? N { get; set; }
+
+    /// <summary>
+    /// Gets or sets the max_tokens option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-max_tokens for more information
+    /// </remarks>
+    public int? MaxTokens { get; set; }
+
+    /// <summary>
+    /// Gets or sets the presence_penalty option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-presence_penalty for more information
+    /// </remarks>
+    public double? PresencePenalty { get; set; }
+
+    /// <summary>
+    /// Gets or sets the frequency_penalty option for chat completion.
+    /// </summary>
+    /// <remarks>
+    /// Refer to https://platform.openai.com/docs/api-reference/chat/create#chat/create-frequency_penalty for more information
+    /// </remarks>
+    public double? FrequencyPenalty { get; set; }
+
+    /// <summary>
+    /// Gets or sets the logit_bias option for chat completion.
+    /// </summary>
+    /// <example>
+    ///     <para>
+    ///         The following is an example showing how to set the property from a string
+    ///     </para>
+    ///     <code>parameters.LogitBias = JsonNode.Parse("{\"50256\": -100}");</code>
```

Comment:
Can we pass also an array to the `JsonNode.Parse` method?

Reply:
So, if the `legit_bias` parameter of the Chat Completion API supports arrays (as it seems), we need to use another type rather than `JsonNode`.

Reply:
From OpenAI documentation I read this "Accepts a json object that maps tokens". So if the string is a valid JSON we have already covered all the cases. Maybe I'm missing something

Reply:
You just told me that we can't pass an array to `JsonNode.Parse`, so for example this isn't supported, right?

JsonNode.Parse("{ [ \"50256\": -100 ], [ \"50257\": -42 ] }");

But OpenAI documentation say that "Accepts a json object that maps TOKENS", so we need to use an array, or not?

Reply:
The JsonNode parser is ok: after some tests on calling the API, I found out the right format to pass more than one token.

```json
{
   "50256": -100,
   "50257": -42
}
```